_includes/payment-order-other-features.md

```diff
@@ -279,6 +279,15 @@ for future purchases. The payer is, by default, asked if they want to store
 their payment details, so even with `generatePaymentToken` set to `true`, it is
 still up to the payer if they want the details stored or not.
 
+Remember that you will have the responsibility to enforce GDPR requirements and
+let the payer remove active payment tokens when they want. It is up to you how
+to implement this functionality on your side but Swedbank Pay has the API you
+need to ensure that cleaning up old data is easy. It is possible to query for
+all active payment tokens registered on a specific `payerReference`. This query
+will also list the operations used for deleting tokens. You can choose to remove
+all tokens for that payer or only a subset of all tokens connected to that
+`payerReference`.
```

Comment:
Convert a part of this paragraph into an alert and then move the alert below further up so they aren't right next to each other.

Reply:
Which part do you want me to convert?

Reply:
The first sentence, perhaps?